api/src/database/index.ts

```diff
@@ -165,14 +162,14 @@ export function getDatabase(): Knex {
 
 		Object.assign(knexConfig, { client: 'mysql2' });
 
-		poolConfig.afterCreate = async (conn: any, callback: any) => {
+		poolConfig.afterCreate = (conn: any, callback: any) => {
 			logger.trace('Retrieving database version');
-			const run = promisify(conn.query.bind(conn));
 
-			const version = await run('SELECT @@version;');
-			databaseVersion = version[0]['@@version'];
+			conn.query('SELECT @@version;', (_err: any, rows: any[], _fields: any) => {
```

Comment:
As [MYSQL 5.7 is EOL](https://endoflife.date/mysql) we can probably remove this query for `databaseVersion` and its relevant usage. This is only used for checking `mysql 5.7` when we generate relational sort queries